docs/v4/CreatingCustomCharacter.md

```diff
@@ -0,0 +1,144 @@
+# Creating a Custom Character
+*Objection 4* makes it very easy to add your own characters to the character
+pool! In this tutorial, I'll go through the steps of adding Athena Cykes.
+
+**NOTE:** You'll need access to the `assets_v4` folder of *Objection* in order to
+add custom characters. If you're using the public social media bots, this won't
+work.
+
+## Setting up the character folder
+Inside of the `characters` folder, create a new folder with a name for your
+character. This will be the character's ID, and the name by which other parts
+of the engine will refer to them. The name can be anything that a folder/file
+can be named, **with the notable exception that it must not start with an
+underscore or else it will be ignored**.
+
+For this example, I'll name the folder `athena`, giving the character that ID.
+
+The folder will also need a `config.toml` file. Copy the file named
+`config_template.toml` from the `characters` folder into your new folder, and
+rename it to `config.toml`. For now, we'll leave it be.
+
+## Getting the sprites
+This is probably the hardest part. Sprites will need to be **animated GIFs**
+with a size of **256x192**. See http://www.court-records.net/sprites1.htm
```

Comment:
We should be able to resize the sprites maybe?

Reply:
It should be trivial to force the image to be 256x192 in the engine. That would still require the sprites to have a 4:3 aspect ratio, though. Any other aspect ratio image would be squashed into those dimensions, and would probably look weird as a result.

Allowing any arbitrarily sized image and ensuring the correct aspect ratio would probably take a lot more work - for those images/animations I think we'd need to allow the user to configure target height/width (and maybe which dimension should be inferred from the other) as well as an origin/pivot point for the image to be drawn from. 

From what I've seen of the official sprite rips and fan edits, it seems somewhat standard to use the 256x192 size, or at least 4:3 aspect ratio. As a result I think maybe keeping the 4:3 requirement for now makes sense, and if people say they'd like to be able to use character sprites with other dimensions we can revisit it.

Reply:
The character sprites are now resized to the 256x192 size by the engine